src/translations.h

```diff
@@ -35,7 +35,13 @@ const char *npgettext( const char *context, const char *msgid, const char *msgid
 
 const char *strip_positional_formatting( const char *msgid );
 
+// If PRINTF_CHECKS is enabled, have to skip the function call so GCC can see the format string
+#if defined(PRINTF_CHECKS) && defined(__GNUC__)
```

Comment:
I could compile the project, I was getting the following error:

```
make LUA=1 TILES=1 RELEASE=1 LOCALIZE=0

/bin/sh: 1: astyle: not found
Cannot run an astyle check, your system either does not have astyle, or it is too old.
Cannot lint JSON, missing usable perl binary and/or p5-JSON module
g++  -o cataclysm-tiles obj/tiles/savegame_legacy.o obj/tiles/armor_layers.o obj/tiles/requirements.o obj/tiles/sdltiles.o obj/tiles/char_validity_check.o obj/tiles/init.o obj/tiles/basecamp.o obj/tiles/event.o obj/tiles/name.o obj/tiles/vehicle_selector.o obj/tiles/crafting_gui.o o
bj/tiles/npcmove.o obj/tiles/artifact.o obj/tiles/bionics.o obj/tiles/npc_class.o obj/tiles/scenario.o obj/tiles/cata_tiles.o obj/tiles/dependency_tree.o obj/tiles/player_activity.o obj/tiles/iuse.o obj/tiles/mapgenformat.o obj/tiles/mondeath.o obj/tiles/catalua.o obj/tiles/monattac
k.o obj/tiles/editmap.o obj/tiles/coordinate_conversions.o obj/tiles/posix_time.o obj/tiles/faction.o obj/tiles/skill.o obj/tiles/weather_data.o obj/tiles/item_stack.o obj/tiles/scent_map.o obj/tiles/tileray.o obj/tiles/mission.o obj/tiles/game.o obj/tiles/version.o obj/tiles/clzone
s.o obj/tiles/gamemode.o obj/tiles/mission_start.o obj/tiles/projectile.o obj/tiles/cursesport.o obj/tiles/mondefense.o obj/tiles/computer.o obj/tiles/messages.o obj/tiles/action.o obj/tiles/mutation_ui.o obj/tiles/line.o obj/tiles/bodypart.o obj/tiles/input.o obj/tiles/color.o obj/
tiles/veh_interact.o obj/tiles/mission_end.o obj/tiles/output.o obj/tiles/mutation_data.o obj/tiles/wincurse.o obj/tiles/main_menu.o obj/tiles/monfaction.o obj/tiles/cata_utility.o obj/tiles/mod_manager_ui.o obj/tiles/submap.o obj/tiles/mission_fail.o obj/tiles/mattack_actors.o obj/
tiles/flag.o obj/tiles/overmap.o obj/tiles/mtype.o obj/tiles/consumption.o obj/tiles/worldfactory.o obj/tiles/creature.o obj/tiles/ranged.o obj/tiles/pickup.o obj/tiles/map.o obj/tiles/item.o obj/tiles/mapsharing.o obj/tiles/itype.o obj/tiles/auto_pickup.o obj/tiles/main.o obj/tiles
/recipe_dictionary.o obj/tiles/overlay_ordering.o obj/tiles/effect.o obj/tiles/savegame.o obj/tiles/morale.o obj/tiles/mission_ui.o obj/tiles/item_search.o obj/tiles/inventory_ui.o obj/tiles/weather_gen.o obj/tiles/pathfinding.o obj/tiles/monstergenerator.o obj/tiles/simplexnoise.o 
obj/tiles/item_factory.o obj/tiles/speech.o obj/tiles/construction.o obj/tiles/gates.o obj/tiles/item_group.o obj/tiles/map_selector.o obj/tiles/translations.o obj/tiles/path_info.o obj/tiles/iuse_software_sokoban.o obj/tiles/live_view.o obj/tiles/tutorial.o obj/tiles/catacharset.o 
obj/tiles/bonuses.o obj/tiles/mapbuffer.o obj/tiles/weather.o obj/tiles/iuse_software_kitten.o obj/tiles/wish.o obj/tiles/bionics_ui.o obj/tiles/debug_menu.o obj/tiles/veh_type.o obj/tiles/iuse_software_snake.o obj/tiles/savegame_json.o obj/tiles/filesystem.o obj/tiles/overmapbuffer
.o obj/tiles/crafting.o obj/tiles/active_item_cache.o obj/tiles/emit.o obj/tiles/sounds.o obj/tiles/mapdata.o obj/tiles/vehicle_group.o obj/tiles/text_snippets.o obj/tiles/npc.o obj/tiles/harvest.o obj/tiles/mutation.o obj/tiles/field.o obj/tiles/ui.o obj/tiles/help.o obj/tiles/crea
ture_tracker.o obj/tiles/material.o obj/tiles/visitable.o obj/tiles/craft_command.o obj/tiles/melee.o obj/tiles/start_location.o obj/tiles/addiction.o obj/tiles/fault.o obj/tiles/mapgen_functions.o obj/tiles/mapgen.o obj/tiles/explosion.o obj/tiles/advanced_inv.o obj/tiles/mod_manag
er.o obj/tiles/player.o obj/tiles/item_action.o obj/tiles/character.o obj/tiles/missiondef.o obj/tiles/mission_place.o obj/tiles/drawing_primitives.o obj/tiles/newcharacter.o obj/tiles/monmove.o obj/tiles/calendar.o obj/tiles/martialarts.o obj/tiles/vehicle.o obj/tiles/debug.o obj/t
iles/item_location.o obj/tiles/activity_handlers.o obj/tiles/defense.o obj/tiles/json.o obj/tiles/vitamin.o obj/tiles/game_inventory.o obj/tiles/trapfunc.o obj/tiles/lightmap.o obj/tiles/safemode_ui.o obj/tiles/dump.o obj/tiles/mission_companion.o obj/tiles/trap.o obj/tiles/npctalk.
o obj/tiles/damage.o obj/tiles/profession.o obj/tiles/iuse_software.o obj/tiles/mongroup.o obj/tiles/animation.o obj/tiles/rng.o obj/tiles/options.o obj/tiles/inventory.o obj/tiles/iuse_actor.o obj/tiles/iuse_software_minesweeper.o obj/tiles/iexamine.o obj/tiles/activity_item_handli
ng.o obj/tiles/lua_console.o obj/tiles/monster.o obj/tiles/ammo.o obj/tiles/turret.o -llua5.2 -L/usr/lib/x86_64-linux-gnu -lSDL2 -lSDL2_ttf -lSDL2_image
obj/tiles/armor_layers.o: In function `(anonymous namespace)::clothing_layer(item const&) [clone .constprop.245]':
armor_layers.cpp:(.text+0x1ee): undefined reference to `strip_positional_formatting(char const*)'
obj/tiles/armor_layers.o: In function `player::sort_armor()':
armor_layers.cpp:(.text+0x6ff): undefined reference to `strip_positional_formatting(char const*)'
armor_layers.cpp:(.text+0x722): undefined reference to `strip_positional_formatting(char const*)'
armor_layers.cpp:(.text+0x746): undefined reference to `strip_positional_formatting(char const*)'
armor_layers.cpp:(.text+0x76a): undefined reference to `strip_positional_formatting(char const*)'
obj/tiles/armor_layers.o:armor_layers.cpp:(.text+0x78e): more undefined references to `strip_positional_formatting(char const*)' follow
collect2: error: ld returned 1 exit status
Makefile:661: recipe for target 'cataclysm-tiles' failed
make: *** [cataclysm-tiles] Error 1
```

After reverting changes in this file it worked just fine. Not sure if it's my setup or a bug, sorry.

_(I also have no idea what I'm doing, not a c dev)_